AnisotropicKuwahara/Filter.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+from multiprocessing import Pool
+
+class Filter:
+
+    def setSrc(self, src):
```

Comment:
is this technically an __init__?

Reply:
nvm, realised we need to have this separate since we populate it when reading the EXR